src/setFocus.ts

```diff
@@ -10,8 +14,9 @@ export const focusOn = (target: HTMLInputElement | HTMLFrameElement) => {
 let guardCount = 0;
 let lockDisabled = false;
 
-export const setFocus = (topNode: HTMLElement, lastNode: HTMLInputElement) => {
+export const setFocus = (topNode: HTMLElement, lastNode: HTMLInputElement, options?: Options) => {
   const focusable = getFocusMerge(topNode, lastNode);
+  const { focusOptions = undefined } = options || {};
```

Comment:
```
const { focusOptions } = options || {};
```
This gives the following TS error
`Initializer provides no value for this binding element and the binding element has no default value.`

Hence, did it this way.